pontoon/base/static/js/main.js

```diff
@@ -671,6 +672,11 @@ $(function() {
     } else {
       ul.find('.no-match').hide();
     }
+  }).on('keydown.search', '.menu input[type=search]', function(e) {
+    // Prevent form submission on Enter
+    if (e.which === 13) {
```

Comment:
nit: I wonder if we could change those codes into something more readable for humans.